projects/view/client/projectProfile/projectProfile.html

```diff
@@ -57,8 +77,99 @@
         <p>{{ description }}</p>
       </div>
     </div>
+</div>
+<!-- TODO wrap content with class=row divs where proper -->
+<!-- START OUT OF SCOPE FOR RELEASE 0.2 -->
+<!-- START Project Positions Content (aka Requests) -->
+<div id="project-positions" class="col-md-12">
+  <div id="project-positions-names">
+    <h2 id="project-position-header">Opening Positions</h2>
+    <span id="project-position-main-description"> If you have the right skillset and expereince, you are more than welcome to help out.</span>
+    <div id="project-position-names-seperator" class="col-md-6">
+      <div id="project-position-lists" class="panel-body">
+        <ul id="project-position-names-list">
+          <!-- Should containt collection list of positions in li with position and local-->
 
+          <li><a href="#" onclick="showPosition();" class="project-position-name-link">Position name.</a> <br /> {{ postalAddress.country }}, {{ postalAddress.city }}</li>
+          <li><a href="#" onclick="showPosition();" class="project-position-name-link">Another position name click should load theese.</a> <br /> {{ postalAddress.country }}, {{ postalAddress.city }}</li>
+        </ul>
+      </div>
+    </div>
+    <div id="project-positions-description-container" class="col-md-6">
+      <div id="project-positions-description-text" class="panel-body panel-default">
```

Comment:
Check [Bootstrap panel](http://getbootstrap.com/components/#panels) documentation for proper markup structure and classes.